radsat-sk/framework/wrappers/RUart.c

```diff
@@ -20,7 +20,7 @@ UARTconfig cameraConfig = {AT91C_US_USMODE_NORMAL | AT91C_US_CLKS_CLOCK | AT91C_
  * @param size: The number of bytes to be sent
  * @return An integer error code. 0 indicates success
  */
-uint32_t uartBlockingTransmit(const uint8_t* data, uint16_t size) {
+uint32_t uartTransmit(const uint8_t* data, uint16_t size) {
 	return UART_write(bus0_uart, data, size);
```

Comment:
Lets add an error check, just to be consistent with the other functions in our code. Same with uartRecieve

Reply:
Added the error check.